revolv/templates/donor/dashboard.html

```diff
@@ -3,79 +3,129 @@
 
 {% block title %}Home{% endblock %}
 
+{% block head %}
```

Comment:
Oh god, it was a terrible decision to split the dashboard up into three separate templates. I'm sorry, my dashboard pull request will fix that.